pkg/kubelet/cm/dra/manager.go

```diff
@@ -40,6 +41,8 @@ type ManagerImpl struct {
 	kubeClient clientset.Interface
 }
 
+const draTimeout = 2 * time.Second
```

Comment:
Actually it might be too short.  
How about 30 seconds, which is the same as `KubeletPreStartContainerRPCTimeoutInSecs`?: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/devicemanager/endpoint.go#L118

Reply:
Would it make sense to have a common constant for the timeout and use it in both DRA and Device Manager?

Reply:
I mean how about setting `draTimeout` as 30 seconds, not directly using `KubeletPreStartContainerRPCTimeoutInSecs` .

```
const draTimeout = 30 * time.Second
```

I mentioned `KubeletPreStartContainerRPCTimeoutInSecs` as a reference of timeout value.

Reply:
```KubeletPreStartContainerRPCTimeoutInSecs``` is a different timeout, I believe.
I was thinking about introducing new constant and using it as a Dial timeout in the  [Device Plugin ](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/devicemanager/plugin/v1beta1/client.go#L124) and DRA Manager.

Reply:
I see.
If the new constant is used in the Device Plugin and DRA Manager, we shouldn't define it in `k8s.io/kubernetes/pkg/kubelet/cm/dra`, right? I think it would be `k8s.io/kubernetes/pkg/kubelet/cm`.

Reply:
> Thank you for your PR. It looks good overall. One general note: would it be better to move this logic to the lower level(pkg/kubelet/plugin/dra/plugin/client.go)?

@Saza-ku what do you think about this?

Reply:
> would it be better to move this logic to the lower level(pkg/kubelet/plugin/dra/plugin/client.go)?

Certainly. It looks like `Client` is responsible for gRPC communication. I'll also fix it.